python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -620,11 +621,15 @@ async def _wait_one_response(
                 # Check if this was an abort/error created by scheduler
                 if isinstance(out["meta_info"].get("finish_reason"), dict):
                     finish_reason = out["meta_info"]["finish_reason"]
-                    if (
-                        finish_reason.get("type") == "abort"
-                        and finish_reason.get("status_code") == HTTPStatus.BAD_REQUEST
+                    finish_type = finish_reason.get("type")
+                    finish_code = finish_reason.get("status_code")
+                    if finish_type == "abort" and (
+                        finish_code == HTTPStatus.BAD_REQUEST
+                        or finish_code == HTTPStatus.FORBIDDEN
                     ):
-                        raise ValueError(finish_reason["message"])
+                        raise CustomReqError(
+                            code=finish_code, message=finish_reason["message"]
+                        )
```

Comment:
same question here. If we could still use `HTTPStatus.BAD_REQUEST`, we won't need to introduce `CustomReqError`